npsat_manager/views.py

```diff
@@ -91,13 +92,77 @@ class ModelRunViewSet(viewsets.ModelViewSet):
 	Test
 
 	Permissions: Must be authenticated
+
+	Optional params:
+		filter:
+			status: all(default) or a int array joined by comma, this will filter status
+		tags:
+			public: true(default), if the user want to include public model
+			isBase: true(default), if the user want to include base model
+			origin: true(default), if the user want to include self-created model
+			scenarios: false(default) ro a int array joined by comma, this will filter scenarios
+		search:
+			search: false(default) or string, this will search the model name and desc
+		sorter:
+			false(default) or formatted string as `{param},{ascend | descend}`
+	These params are additional filter to sift models to return the model list
 	"""
 	permission_classes = [IsAuthenticated]
 
 	serializer_class = serializers.RunResultSerializer
 
 	def get_queryset(self):
```

Comment:
Let's get a test committed for this function to make sure that the permissions behave - seems like a function we might modify in the future where we could accidentally expose all model runs to everyone. Looks good otherwise

Reply:
Permissions should be good. It will first authenticate the user, then gets the user which kinds of models based on the params.

I will push a test case with the next commit.